### ISSUE TITLE
Execute function from bracket prop property access

### ISSUE THREAD

**Issue by jcubic** [jcubic/gaiman#41]
Add syntax:

```ruby
let list = {
   hello => lambda()
     echo "HELLO FROM LAMBDA"
   end
}
list["hello"]()
```

This is important if "hello" is variable:

```ruby
let cmd = ask "? "
if type list[cmd] == "function" then
   list[cmd]()
end
```

**Comment by jcubic** [comment]
The call and methods should be refactored. It should support:
``ruby
list[cmd](10)()
list[cmd](10).foo.bar()
list[cmd](10).foo.bar()()
```

**Comment by jcubic** [comment]
Original issue is fixed by 781de690a4ec46f98290e18820d151b22659bf3e the rest needs some work.
